src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -783,7 +783,19 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_conv_bias_bprop()
             {
                 if (std::dynamic_pointer_cast<op::Sum>(delta_user))
                 {
-                    auto bias_shape = delta_user->get_output_shape(0);
+                    auto bias = std::dynamic_pointer_cast<op::Sum>(delta_user);
+                    auto bias_shape = bias->get_shape();
+                    if (bias_shape.size() > 1)
+                    {
+                        NGRAPH_DEBUG
+                            << "mpattern = " << m.get_match_root()->get_name()
+                            << "conv_bias bias shape != 1, requires reshape to match filter count.";
+                        ngraph::AxisVector order(bias_shape.size());
+                        std::iota(begin(order), end(order), 0);
+                        auto bias_reshape = std::make_shared<op::Reshape>(
+                            bias, order, Shape{conv_bprop->get_filters_shape()[0]});
+                        bias_shape = bias_reshape->get_shape();
```

Comment:
Cant we compute the correct shape without creating an op?

Reply:
I think the shape would be `conv_bprop->get_filters_shape()[0]` we probably need a reshape because of a check in `ConvolutionBiasBackprop` which means we might be doing a copy (for Reshape) even though we could avoid one if `ConvolutionBiasBackprop` was a little bit more flexible and just checked for `shape_size(bias) ==  conv_bprop->get_filters_shape()[0]}`

Reply:
@jbobba i misread that code... the right shape always seems to be `Shape{conv_bprop->get_filters_shape()[0]}` @nbpatel ?